ui/app/mainui/AppMain.qml

```diff
@@ -218,14 +218,111 @@ Item {
+                    let toastType = Constants.ephemeralNotificationType.normal
+                    let icon = ""
+                    if(error) {
+                        toastTitle = qsTr("Failed to set spending cap: %1 in %2 for %3 on %4").arg(approvalAmount).arg(fromAccountName).arg(Constants.swap.paraswapUrl).arg(networkName)
+                        toastSubtitle = ""
+                        urlLink = ""
+                        toastType = Constants.ephemeralNotificationType.danger
+                        icon = "warning"
+                    }
+                    Global.displayToastMessage(toastTitle, toastSubtitle, icon, !error, toastType, urlLink)
+                }
+                break
+            case Constants.SendType.Swap:
+                let fromToken_ = ModelUtils.getByKey(appMain.tokensStore.plainTokensBySymbolModel, "key", fromTokenKey)
+                let toToken = ModelUtils.getByKey(appMain.tokensStore.plainTokensBySymbolModel, "key", toTokenKey)
+                let fromAccountName_ = ModelUtils.getByKey(appMain.transactionStore.accounts, "address", fromAddr, "name")
+                let networkName_ = ModelUtils.getByKey(WalletStore.RootStore.filteredFlatModel, "chainId", chainId, "chainName")
+                if(!!fromToken_ && !!toToken && !!fromAccountName_ && !!networkName_) {
+                    let fromSwapAmount = currencyStore.formatCurrencyAmountFromBigInt(fromAmount, fromToken_.symbol, fromToken_.decimals)
+                    let toSwapAmount = currencyStore.formatCurrencyAmountFromBigInt(toAmount, toToken.symbol, toToken.decimals)
+                    let toastTitle_ = qsTr("Swapping %1 to %2 in %3 using %4 on %5").arg(fromSwapAmount).arg(toSwapAmount).arg(fromAccountName_).arg(Constants.swap.paraswapUrl).arg(networkName_)
+                    let toastSubtitle_ = qsTr("View on %1").arg(networkName_)
+                    let urlLink_ = "%1/%2".arg(appMain.rootStore.getEtherscanLink(chainId)).arg(txHash)
+                    let toastType_ = Constants.ephemeralNotificationType.normal
+                    let icon_ = ""
+                    if(error) {
+                        toastTitle_ = qsTr("Failed to swap %1 to %2 in %3 using %4 on %5").arg(fromSwapAmount).arg(toSwapAmount).arg(fromAccountName_).arg(Constants.swap.paraswapUrl).arg(networkName_)
+                        toastSubtitle_ = ""
+                        urlLink_ = ""
+                        toastType_ = Constants.ephemeralNotificationType.danger
+                        icon_ = "warning"
+                    }
+                    Global.displayToastMessage(toastTitle_, toastSubtitle_, icon_, !error, toastType_, urlLink_)
+                }
+                break
+            default:
+                if (!error) {
+                    Global.displayToastMessage(qsTr("Transaction pending..."),
+                                               qsTr("View on etherscan"),
```

Comment:
Is the name always "etherscan"? :)

Reply:
aaah was already there and missed seeing it, fixed now! :)